lib/google-blog-feed-loader.js

```diff
@@ -0,0 +1,91 @@
+var FEED_URL = 'https://blog.webmaker.org/tag/teachtheweb/feed';
+var DEFAULT_CONTENT_SNIPPET_WORD_COUNT = 70;
+var SCRIPT_SRC = 'https://www.google.com/jsapi';
+
+// http://stackoverflow.com/a/822464
+var HTML_TAG_RE = /<(?:.|\n)*?>/gm;
+
+var scriptLoading = false;
+var scriptLoadCallbacks = [];
+
+function handleScriptLoad() {
```

Comment:
Awsm I will do that! Heading out for the weekend but will do it first thing monday :)